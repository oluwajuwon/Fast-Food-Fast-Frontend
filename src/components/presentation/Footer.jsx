import React from 'react';

const Footer = () => (
  <footer>
    <div className="text-center white-text">
      Fast-Food-Fast &copy; 2018 &nbsp; Developed by &nbsp;
      <b>
        Juwon.
      </b>
    </div>
  </footer>
);

export default Footer;
