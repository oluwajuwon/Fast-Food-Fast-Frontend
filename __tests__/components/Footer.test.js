import React from 'react';
import { render } from 'react-testing-library';
import Footer from '../../src/components/presentation/Footer';

describe('Test for footer component', () => {
  const { container } = render(<Footer />);
  it('Should contain a particular div block', () => {
    expect(container).toMatchSnapshot();
  });
});
