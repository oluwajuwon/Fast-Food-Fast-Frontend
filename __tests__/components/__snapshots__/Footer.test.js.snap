// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for footer component Should contain a particular div block 1`] = `
<footer>
  <div
    className="text-center white-text"
  >
    Fast-Food-Fast © 2018   Developed by  
    <b>
      Juwon.
    </b>
  </div>
</footer>
`;
